### ISSUE TITLE
Python: TypeError for Client.invoke() without xml_doc specified

### ISSUE THREAD

**Issue by dcoshea** [Openwsman/openwsman#59]
http://openwsman.github.io/rdoc/Openwsman/Client.html documents these signatures for `invoke()`:

invoke(options, uri, "method-name") → XmlDoc
invoke(options, uri, "method-name", xml_doc) → XmlDoc 

With openwsman-python-2.4.11-132.1 installed from http://download.opensuse.org/repositories/Openwsman/CentOS_CentOS-6/Openwsman.repo, I was able to call `invoke()` without `xml_doc` specified, but after upgrading to the latest version openwsman-python-2.4.11-132.1, I get an error:

```
[...]
  File "/usr/lib64/python2.6/site-packages/pywsman.py", line 3096, in invoke
    return _pywsman.Client_invoke(self, options, resource_uri, method, data)
TypeError: in method 'Client_invoke', argument 5 of type 'WsXmlDocH'
```

I can work around this by passing a dummy value for `xml_doc`, e.g. `pywsman.XmlDoc("dummy")`. 

`/usr/lib64/python2.6/site-packages/pywsman.py` line 3094 contains:
```
    def invoke(self, options, resource_uri, method, data=0):
```

If I change the `0` to `None`, this seems to fix the issue.

There seem to be quite a few other instances of `=0` in that file, though.

**Comment by kkaempf** [comment]
Argh, that's a SWIG bug, thanks for noticing !

**Comment by kkaempf** [comment]
Will be fixed in SWIG 3.0.6
